File: DigitalHouseSystemApi/Models/Tariff.cs
namespace DigitalHouseSystemApi.Models
{
    public class Tariff
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int InternetSpeed { get; set; }
        public string ConversationTime { get; set; }
        public string SMS { get; set; }  
        public int? RoamingInternet { get; set; }
        public int? InternationalNetworkCalls { get; set; }
        public bool E_bill { get; set; }
        public decimal Price { get; set; }
        public decimal? Discount { get; set; }


        public ICollection<Magenta1Tariff>? Magenta1Tariffs { get; set; }

        public int TariffTypeId { get; set; }
        public TariffType TariffType { get; set; }
    }
}

File: client/src/app/_models/tariff.ts
export interface Tariff{
    id:number,
    name: string,
    internetSpeed: number,
    conversationTime: string,
    sms: string,
    roamingInternet?: number,
    internationalNetworkCalls?: number,
    e_bill?: boolean,
    price: number,
    discount?: number
}